src/input_xml.F90

```diff
@@ -5879,14 +5848,13 @@ subroutine read_ce_cross_sections(libraries, library_dict)
       end do
     end do
 
-    n_nuc_zaid_total = index_nuc_zaid
-
     ! Associate S(a,b) tables with specific nuclides
     call assign_sab_tables()
 
     ! Show which nuclide results in lowest energy for neutron transport
     do i = 1, size(nuclides)
-      if (nuclides(i) % energy(nuclides(i) % n_grid) == energy_max_neutron) then
+      if (nuclides(i) % grid(1) % energy(size(nuclides(i) % grid(1) % energy)) &
```

Comment:
its also probably correct to go through each temperature here..... but i dont know that it matters much.  (Does NJOY pin the bottom energy point or is it fair game to remove in thinning?